modules/nf-core/tcoffee/consensus/tests/main.nf.test

```diff
@@ -0,0 +1,132 @@
+nextflow_process {
+
+    name "Test Process TCOFFEE_CONSENSUS"
+    script "../main.nf"
+    process "TCOFFEE_CONSENSUS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "tcoffee"
+    tag "tcoffee/consensus"
+    tag "tcoffee/align"
+    tag "pigz"
+    tag "famsa/guidetree"
+    tag "famsa/align"
+
+    config "./sequence.config"
+
+    setup {
+        run("FAMSA_GUIDETREE") {
+            script "../../../famsa/guidetree/main.nf"
+            process {
+                """
+                input[0] = [ [ id:'test' ],
+                                file(params.modules_testdata_base_path + "../../multiplesequencealign/testdata/setoxin-ref.fa", checkIfExists: true)                                 
+                            ]
+                """
+            }
+        }
+        run("FAMSA_ALIGN") {
+            script "../../../famsa/align/main.nf"
+            process {
+                """
+                input[0] = [ [ id:'test' ],
+                                file(params.modules_testdata_base_path + "../../multiplesequencealign/testdata/setoxin-ref.fa", checkIfExists: true)                                 
+                            ]
+                input[1] = [[:],[]]
+                input[2] = false
+
+                """
+            }
+        }
+        run("TCOFFEE_ALIGN") {
+            script "../../../tcoffee/align/main.nf"
+            process {
+                """
+                input[0] = [ [ id:'test' ],
+                                file(params.modules_testdata_base_path + "../../multiplesequencealign/testdata/setoxin-ref.fa", checkIfExists: true)                                 
+                            ]
+                input[1] = [[:],[]]
+                input[2] = [[:],[],[]]
+                input[3] = false
+
+                """
+            }
+        }          
+    }
+
+    test("consensus - no tree - uncompressed - seatoxin ") {
+       
+        when {
+            process {
+                """
+                msas = FAMSA_ALIGN.out.alignment.mix(TCOFFEE_ALIGN.out.alignment).groupTuple()
+                input[0] = msas
+                input[1] = [[:],[]]
+                input[2] = false
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                                              process.out.alignment,
+                                              process.out.versions
+                                              ).match()
+                },
```

Comment:
Sorry I made the indents a bit weird when I suggested this. I would remove some of the spaces so that it aligns nicely to the `snapshot(` part 🙈

Reply:
done :)